ncm-spma/src/main/perl/spma.pod

```diff
@@ -11,9 +10,18 @@ NCM::Component::spma - Quattor's package manager
 
 The SPMA component passes off to different back-ends depending
 on the type of packaging system specified in
-B</software/components/spma/packager>.
+C<< /software/components/spma/packager >>.
+
+Currently available:
+
+=over
+
+=item * B<NCM::Component::spma::ips> for Solaris.
```

Comment:
These ones are better served as `C<NCM::Component::spma::ips>` and the like (code blocks instead of bold).